apps/aecore/lib/aecore/account/account_state_tree.ex

```diff
@@ -44,6 +46,23 @@ defmodule Aecore.Account.AccountStateTree do
     put(tree, key, fun.(get(tree, key)))
   end
 
+  @spec safe_update(
```

Comment:
same question as above, why do we need this function here?

Reply:
Same as apply_transfer